agents-api/agents_api/common/retry_policies.py

```diff
@@ -0,0 +1,18 @@
+from datetime import timedelta
+
+from temporalio.common import RetryPolicy
+
+DEFAULT_RETRY_POLICY = RetryPolicy(
+    initial_interval=timedelta(seconds=1),
+    backoff_coefficient=2,
+    maximum_attempts=2,
+    maximum_interval=timedelta(seconds=10),
```

Comment:
this should be configurable based on testing or debug -- reduce to 10 for that otherwise, 300